apps/web/src/components/invoice-generator/index.tsx

```diff
@@ -239,6 +245,14 @@ function InvoiceItem({
 }) {
   const t = useScopedI18n("invoicegenerator");
 
+  const [quantity, setQuantity] = useState(0);
+  const [unitPrice, setUnitPrice] = useState(0);
+  const [totalPrice, setTotalPrice] = useState(0);
+
+  useEffect(() => {
+    setTotalPrice(quantity * unitPrice);
+  }, [quantity, unitPrice]);
+
   return (
     <fieldset>
```

Comment:
Not part of this PR, but I think this fieldset nesting is a bit suspicious, might be worth taking a look at some point

Reply:
i dont know im just a dog